### ISSUE TITLE
[$250] E/App issue for payment of E/E Refactor Policy_Delete #215184

### ISSUE THREAD

**Issue by mallenexpensify** [Expensify/App#10803]
Details in https://github.com/Expensify/Expensify/issues/215184
and
https://github.com/Expensify/App/pull/10491

Adding the `ContributorPlusReview` to assign C+ to review 10491, then will create job and pay once complete

**Comment by mallenexpensify** [comment]
@maddylewis sorry to add/remove ya by mistake
@marcaaron , no action needed,  unassigned ya
@mananjadhav can you please review https://github.com/Expensify/App/pull/10491 and comment in this issue to confirm you're able to do the review? Thx

Job
https://www.upwork.com/jobs/~018fd97fd6e6639650, I hired you Manan, job is private

**Comment by mananjadhav** [comment]
@mallenexpensify Unable to access this URL. https://github.com/Expensify/Expensify/issues/215184 and hence don't have the context for the issue? I'll review the code styling and other stuff meanwhile and try to gauge from the test cases but access to the issue will be of help.

**Comment by aldo-expensify** [comment]
@mananjadhav there is still a couple of PRs that need to be merged before you can test this against the API. I'll remove the HOLD in the title when those other PRs has been merged/deployed. You can meanwhile review the code if you want.

**Comment by mallenexpensify** [comment]
@aldo-expensify, should this be on hold? Not seeing in title

**Comment by aldo-expensify** [comment]
I don't think so. I was talking about the HOLD in the PRs that depend on other PRs, no this GH issue.

**Comment by mallenexpensify** [comment]
Actively being worked on

**Comment by aldo-expensify** [comment]
@mallenexpensify the internal GH issue https://github.com/Expensify/Expensify/issues/215184 has been closed and the last [PR on App](https://github.com/Expensify/App/pull/10491) has been reviewed/merged/deployed. Not sure what is the next step here.

**Comment by mallenexpensify** [comment]
Thanks @aldo-expensify , Paid $250 to @mananjadhav for the review. 
If either of you think that amount should be different, please comment
